lib/spack/spack/modules.py

```diff
@@ -327,6 +327,9 @@ def blacklisted(self):
         blacklist_matches = [x
                              for x in configuration.get('blacklist', [])
                              if self.spec.satisfies(x)]
+        blacklisted_as_implicit = (configuration.get('blacklist_implicits') and
```

Comment:
Ok, though I didn't buy into the `need_to_blacklist` name (it feels like it overlaps with the other/older blacklisting bits.